src/test/java/org/codeseoul/event_member_management/rsvp/RsvpRestIntegrationTests.java

```diff
@@ -0,0 +1,196 @@
+        aMember = memberRepository.saveAndFlush(new Member(
+                "user",
+                "displayName",
+                "example@email.com",
+                "01234567890",
+                "http://image.com",
+                "About user"));
+        anEvent = eventRepository.save(new Event(
+                "New Event",
+                "such new, so event",
+                Timestamp.valueOf("2022-06-22 08:51:00"),
+                120,
+                "http://image.com",
+                "WCoding",
+                null,
+                null));
+        anRsvp = new Rsvp(aMember, anEvent, rsvpYesState);
+        rsvpRepository.save(anRsvp);
+    }
+
+    @AfterEach
+    void tearDown() {
+        rsvpRepository.deleteAll();
+        rsvpStateRepository.deleteAll();
+        eventRepository.deleteAll();
+        memberRepository.deleteAll();
+    }
+
+    private Member createAnotherMember() {
+        return new Member(
+                "anotherUser",
+                "Another Display Name",
+                "anotherExample@email.com",
+                "2345678901",
+                "http://image.com",
+                "About another user");
+    }
+
+    private Event createAnotherEvent() {
+        return new Event(
+                "Another Event",
+                "This is madness",
+                Timestamp.valueOf("2022-06-23 08:51:00"),
+                120,
+                "https://imgs.com",
+                "Under da sea",
+                null,
+                null
+        );
+    }
+
+    @Test
+    void returnsNotFoundErrorWhenRsvpNotFound() throws Exception {
+        mockMvc.perform(get(String.format("/rsvps/%d", anRsvp.getId() + 1)))
+                .andExpect(status().isNotFound());
+    }
+
+    @Test
+    void findsRspvById() throws Exception {
+        mockMvc.perform(get(String.format("/rsvps/%d", anRsvp.getId())))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.eventTitle", is(anEvent.getTitle())))
+                .andExpect(jsonPath("$.memberDisplayName", is(aMember.getDisplayName())));
+    }
+
+    @Test
+    void returnsNotFoundErrorWhenUpdatingNonExistingRsvp() throws Exception {
+        mockMvc.perform(put(String.format("/rsvps/%d", anRsvp.getId() + 1))
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .content(mapper.writeValueAsString(anRsvp)))
+                .andExpect(status().isNotFound());
+    }
+
+    @Test
+    void updatesAnRsvp() throws Exception {
+        anRsvp.setState(rsvpNoState);
+
+        mockMvc.perform(put(String.format("/rsvps/%d", anRsvp.getId()))
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .content(mapper.writeValueAsString(anRsvp)))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.rsvpState", is(rsvpNoState.getState())));
+    }
```

Comment:
We don't need to worry about it for this PR, but I just realized we should also be testing that the data changes were persisted. The purpose of these endpoints is twofold: change data and inform the caller of results.